src/taipy/core/config/__init__.py

```diff
@@ -105,5 +105,5 @@
 _Checker.add_checker(_CoreSectionChecker)
 _Checker.add_checker(_DataNodeConfigChecker)
 _Checker.add_checker(_TaskConfigChecker)
-_Checker.add_checker(_PipelineConfigChecker)
+_Checker.add_checker(_SequenceConfigChecker)
```

Comment:
Same for the checker, we should have removed it already right?

Reply:
same answer as above